src/runtime/threading_backend.cc

```diff
@@ -26,30 +30,48 @@ class ThreadGroup::Impl {
     for (int i = exclude_worker0; i < num_workers_; ++i) {
       threads_.emplace_back([worker_callback, i] { worker_callback(i); });
     }
+    InitSortedOrder();
+  }
+  ~Impl() { Join(); }
+
+  void Join() {
+    for (auto& t : threads_) {
+      if (t.joinable()) t.join();
+    }
+  }
+
+  int Configure(int mode, int nthreads, bool exclude_worker0) {
+    int num_workers_used = 0;
+    if (mode == -1) {
+      num_workers_used = little_count_;
+    } else if (mode == 1) {
+      num_workers_used = big_count_;
+    } else {
+      // use default
+      num_workers_used = threading::MaxConcurrency();
+    }
+    // if a specific number was given, use that
+    if (nthreads) {
+      num_workers_used = nthreads;
+    }
     const char *val = getenv("TVM_BIND_THREADS");
     if (val == nullptr || atoi(val) == 1) {
```

Comment:
// Skip if sorted_order.size() is bigger than threads_.size()
if (sorted_order_.size() > threads_.size() || not_bind) skip set affinity.